src/farcasterd/runtime.rs

```diff
@@ -469,69 +261,8 @@ impl Runtime {
                 trace!("Hello farcasterd from {}", source);
                 // Ignoring; this is used to set remote identity at ZMQ level
             }
-
-            // 1st protocol message received through peer connection, and last
-            // handled by farcasterd, receiving taker commit because we are
-            // maker
-            (
-                Request::Protocol(Msg::TakerCommit(request::TakeCommit {
-                    commit: _,
-                    public_offer,
-                    swap_id,
-                })),
-                ServiceId::Peer(_),
-            ) => {
-                // public offer gets removed on LaunchSwap
-                if !self.public_offers.contains(&public_offer) {
-                    warn!(
-                        "Unknown (or already taken) offer {}, you are not the maker of that offer (or you already had a taker for it), ignoring it",
-                        &public_offer
-                    );
-                } else {
-                    trace!(
-                        "Offer {} is known, you created it previously, engaging walletd to initiate swap with taker",
-                        &public_offer
-                    );
-                    if let Some(arb_addr) = self.arb_addrs.remove(&public_offer.id()) {
-                        let btc_addr_req =
-                            Request::BitcoinAddress(BitcoinAddress(*swap_id, arb_addr));
-                        endpoints.send_to(
-                            ServiceBus::Msg,
-                            self.identity(),
-                            ServiceId::Wallet,
-                            btc_addr_req,
-                        )?;
-                    } else {
-                        error!("missing arb_addr")
-                    }
-                    if let Some(acc_addr) = self.acc_addrs.remove(&public_offer.id()) {
-                        let xmr_addr_req =
-                            Request::MoneroAddress(MoneroAddress(*swap_id, acc_addr));
-                        endpoints.send_to(
-                            ServiceBus::Msg,
-                            self.identity(),
-                            ServiceId::Wallet,
-                            xmr_addr_req,
-                        )?;
-                    } else {
-                        error!("missing acc_addr")
-                    }
-                    info!("passing request to walletd from {}", source);
-                    self.peerd_ids
-                        .insert(public_offer.offer.id(), source.clone());
-
-                    endpoints.send_to(
-                        ServiceBus::Msg,
-                        self.identity(),
-                        ServiceId::Wallet,
-                        request,
-                    )?;
-                }
-                return Ok(());
-            }
             _ => {
```

Comment:
I find the resulting `match` arms a bit weird now, the main part is in the `_` catch all and we don't use the source anymore